### DISCUSSION TITLE
Is there a way to delete the currently playing file from within smplayer?

### DISCUSSION THREAD

**Discussion by rwernst** [smplayer-dev/smplayer#359] [Q&A] [answered]
Is there a way to delete the currently playing file from within smplayer?

I don't think so but thought I'd ask to see if I was missing something.

I ask as I have a lot of video files from a security camera at my house, some fairly small, that I need to go through and would like to delete those that have nothing to show.

Thanks.

**Comment by smplayer-dev** [comment, marked as answer]
You can delete files from the playlist. But first you need to enable this feature in Preferences -> Playlist -> Misc.

**Reply by rwernst** [reply]
Thanks but I want the file actually deleted from the computer, not just a playlist.

**Reply by smplayer-dev** [reply]
Sorry, I didn't explain well. If you enable that option, when you right click on a file in the playlist you will have the option to remove the file from your computer ("delete file from disk").

**Reply by rwernst** [reply]
Ah, excellent, thanks, that's going to make this a LOT easier!  What an amazing program!

**Reply by NirmalBhasarkar2** [reply]
This feature doesn't work in full screen mode and it directly deletes the video. I need it to send it to recycle bin instead.

**Comment by eximi1** [comment]
Can I request to implement for multiple files be selected to delete them from disk in one go from the playlist?
Currently if you try this, it only deletes one of the files.

**Comment by Alejandrovdv** [comment]
Hola a todos.
Tengo un problema con la accion de eliminar archivo del actual video que se esta reproduciendo.
encontre que se puede habilitar la accion y mapie la tecla Del.
Al principio pense que no funcionaba porque lo hacia desde el mismo reproductor.
Me acabo de dar cuenta que si me posiciono desde la playlist si funciona.
Hay posibilidad de que no tenga que estar activo en la playlist para poder ejecutar la accion?
Gracias
